api/api.go

```diff
@@ -211,6 +212,10 @@ type VersionResponse struct {
 	RequireClientAuthentication bool   `json:"requireClientAuthentication,omitempty"`
 }
 
+// CapabilitiesResponse is the response object that returns the version of the
+// server.
+type CapabilitiesResponse authority.Capabilities
```

Comment:
I think it's safer to make this a new struct instead of an alias. It keeps a cleaner separation between the API and the authority packages and won't "leak" new properties added to `authority.Capabilities` unintentionally.

Reply:
The way you recommend is the way we've done it for other types - like Version I think. So it makes sense to be consistent. But I hate that I'll have to remember to update >1 types. Makes it harder on developers in my opinion. If the general consensus is to go with a new type, then I'm happy to switch.